Client/mods/deathmatch/logic/luadefs/CLuaGUIDefs.cpp

```diff
@@ -281,6 +287,7 @@ void CLuaGUIDefs::AddGuiElementClass(lua_State* luaVM)
     lua_classvariable(luaVM, "text", "guiSetText", "guiGetText");
     lua_classvariable(luaVM, "size", "guiSetSize", "guiGetSize");
     lua_classvariable(luaVM, "position", "guiSetPosition", "guiGetPosition");
+    //lua_classvariable(luaVM, "chatboxCharacterLimit", "setChatboxCharacterLimit", "getChatboxCharacterLimit");
```

Comment:
Why is this commented out?

Reply:
no good reason that I can think of, probably just forgot to uncomment